WooCommerce/Classes/ViewRelated/Products/Add Product/AddProductWithAI/StartingInfo/ProductCreationAIStartingInfoView.swift

```diff
@@ -16,95 +16,117 @@ struct ProductCreationAIStartingInfoView: View {
     }
 
     var body: some View {
-        ScrollView {
-            VStack(alignment: .leading, spacing: Layout.parentSpacing) {
-                VStack(alignment: .leading, spacing: Layout.titleBlockSpacing) {
-                    // Title label.
-                    Text(Localization.title)
-                        .fontWeight(.bold)
-                        .titleStyle()
-
-                    // Subtitle label.
-                    Text(Localization.subtitle)
-                        .foregroundStyle(Color(.secondaryLabel))
-                        .bodyStyle()
-                }
-
-                VStack(alignment: .leading, spacing: Layout.textFieldBlockSpacing) {
-                    VStack(alignment: .leading, spacing: Layout.editorBlockSpacing) {
-                        VStack(spacing: 0) {
-                            ZStack(alignment: .topLeading) {
-                                TextEditor(text: $viewModel.features)
-                                    .bodyStyle()
-                                    .foregroundStyle(.secondary)
-                                    .padding(insets: Layout.messageContentInsets)
-                                    .frame(minHeight: Layout.minimumEditorHeight, maxHeight: .infinity)
-                                    .focused($editorIsFocused)
-
-                                // Placeholder text
-                                placeholderText
-                            }
+        ScrollViewReader { proxy in
+            ScrollView {
+                VStack(alignment: .leading, spacing: Layout.parentSpacing) {
+                    VStack(alignment: .leading, spacing: Layout.titleBlockSpacing) {
+                        // Title label.
+                        Text(Localization.title)
+                            .fontWeight(.bold)
+                            .titleStyle()
+
+                        // Subtitle label.
+                        Text(Localization.subtitle)
+                            .foregroundStyle(Color(.secondaryLabel))
+                            .bodyStyle()
+                    }
 
-                            Divider()
-                                .frame(height: Layout.dividerHeight)
-                                .foregroundColor(Color(.separator))
 
-                            switch viewModel.imageState {
-                            case .empty:
-                                readTextFromPhotoButton
-                                    .padding(insets: Layout.readTextFromPhotoButtonInsets)
+                    VStack(alignment: .leading, spacing: Layout.textFieldBlockSpacing) {
+                        VStack(alignment: .leading, spacing: Layout.editorBlockSpacing) {
+                            VStack(spacing: 0) {
+                                ZStack(alignment: .topLeading) {
+                                    TextEditor(text: $viewModel.features)
+                                        .id("TextEditor")
```

Comment:
Nit: We should keep the ID a constant to avoid mistyping it somewhere else.

Reply:
Good suggestion, done here a2bd8e29da1c2b38db2f3612a7210e35bd6f938a